tests/unittests/execution_store_service_tests.py

```diff
@@ -0,0 +1,396 @@
+"""
+FiftyOne execution store related unit tests.
+
+| Copyright 2017-2024, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+
+import pytest
+
+import fiftyone as fo
+from fiftyone.operators.store import ExecutionStoreService
+
+from decorators import drop_collection, drop_datasets
+
+TEST_COLLECTION_NAME = "execution_store_test_collection"
+
+
+@pytest.fixture
+def dataset():
+    return fo.Dataset(name="test_dataset")
```

Comment:
could just use `fo.Dataset()` here to ensure this test won't fail if someone happens to have a pre-existing `test_dataset`

Reply:
removed `name=`